packages/lesmis/src/views/AdminPanel/SurveyResponsesView.js

```diff
@@ -49,37 +55,19 @@ const COLUMNS = [
   {
     Header: 'Reject',
     source: 'id',
+    Cell: RejectButton,
     type: 'delete',
```

Comment:
Can this type be dropped now?

Reply:
actually the type brings in the styling for the column so I will leave it as is